routes/categories.ts

```diff
@@ -0,0 +1,12 @@
+var express = require('express');
+var router = express.Router();
+var category = require('../models/category');
+
+/* GET categories. */
+router.get('/', function(req, res, next) {
```

Comment:
This is not accessible from an HTML app. 
Needs code such as   <code>res.header("Access-Control-Allow-Origin", "*");</code> (can insert your origin there ...)